src/PaymentGateways/PayPalStandard/Actions/RedirectOffsitePayment.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Give\PaymentGateways\PayPalStandard\Actions;
+
+use Give\Helpers\Form\Utils;
+use Give\Session\SessionDonation\DonationAccessor;
+
+/**
+ * This class handles after offsite payment redirect.
+ *
+ * @unreleased
+ */
+class RedirectOffsitePayment
```

Comment:
In regard to naming Action classes, I would suggest using a different name, since the `RedirectOffsitePayment` action is not itself redirecting offsite for payment. Instead, maybe this is a `RedirectUrlFactory` or something like that?

Reply:
This should be two separate actions. An Action is just that, a single action. Looks like this would be:
- GenerateDonationReceiptPageUrl
- GenerateDonationFailedPageUrl

Reply:
@kjohnson fixed: https://github.com/impress-org/givewp/compare/f961a2a7a6a2...f4c7843ed9b9